src/utils/postDiscussion.js

```diff
@@ -1,10 +1,10 @@
-export const postNewDiscussion = (title, body) => {
+export const postNewDiscussion = (title, body, tagTitle) => {
   fetch('http://localhost:3000/api/v1/discussions', {
     method: 'POST',
     headers: {
       'Content-Type': 'application/json',
     },
-    body: JSON.stringify({ title, body }),
+    body: JSON.stringify({ title, body, tagTitle }),
```

Comment:
Great that you added this to the post. I am working on rendering the tagtitle in the body of the new discussion.